apps/macOS/FluentUIReactNativeTester_macOS/Config/FluentUIReactNative_debug.xcconfig

```diff
@@ -3,7 +3,22 @@
 //  Licensed under the MIT License.
 //
 
+#include "FluentUIReactNative_common.xcconfig"
 #include "Pods/Target Support Files/Pods-FluentUIReactNativeTester_macOS/Pods-FluentUIReactNativeTester_macOS.debug.xcconfig"
 
 SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG
 SWIFT_OPTIMIZATION_LEVEL = -Onone
+GCC_OPTIMIZATION_LEVEL = 0
+COPY_PHASE_STRIP = NO;
+SWIFT_TREAT_WARNINGS_AS_ERRORS = YES
+CLANG_ANALYZER_NONNULL = YES
+CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE
+CLANG_WARN_BOOL_CONVERSION = YES
+CLANG_WARN_CONSTANT_CONVERSION = YES
+CLANG_WARN_INFINITE_RECURSION = YES
+CLANG_WARN_INT_CONVERSION = YES
+CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE
+CLANG_WARN_UNREACHABLE_CODE = YES
+CLANG_WARN__DUPLICATE_METHOD_MATCH = YES
+GCC_WARN_UNUSED_FUNCTION = YES
+GCC_WARN_UNUSED_VARIABLE = YES
```

Comment:
Why wouldn't any of these also apply to release? Do they stop working with optimization?

Particularly interesting is unused function/variable warnings, which could easily be distinct in release vs. debug with debug only code

Reply:
Debug should catch all of these issues so release can just build and run without extra checks running in the background

Reply:
If we expect to have debug or release macros then it makes sense to move some of these to common. I wasn't planning on that, but I suppose it's low harm even if we don't